package.json

```diff
@@ -14,11 +14,15 @@
   },
   "dependencies": {
     "autoprefixer-loader": "^3.1.0",
-    "babel-core": "^5.8.34",
-    "babel-loader": "^5.4.0",
+    "babel-core": "^6.3.15",
+    "babel-loader": "^6.2.0",
     "babel-plugin-react-display-name": "^2.0.0",
-    "babel-plugin-react-transform": "^1.1.1",
-    "babel-runtime": "^5.8.34",
+    "babel-plugin-react-transform": "^2.0.0-beta1",
+    "babel-plugin-transform-runtime": "^6.3.13",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-react": "^6.3.13",
+    "babel-preset-stage-0": "^6.3.13",
+    "babel-runtime": "^6.3.19",
```

Comment:
It seems like every preset plugin uses `"^5.0.0"` for this dependency - using the same version cuts out a bunch of duplication in `node_modules`